docs/proposals/volume-provisioning.md

```diff
@@ -101,6 +98,12 @@ We propose that:
 
 ### Controller workflow for provisioning volumes
 
+0. Kubernetes administator can configure name of a default StorageClass. This
+   StorageClass instance is then used when user requests a dynamically
+   provisioned volume, but does not specify a StorageClass. In other words,
+   `claim.Spec.Class == ""`
```

Comment:
the use case for secret is along the lines of control API credentials used during the creation of the asset (cloud credentials, admin RBD secret, login to fibre channel provisioning API etc..)  a secret as a parameter to provisioner used to encrypt the volume or similar would result in every volume created by that provisioner using the same secret. 

a better use case is the provisioner creating a unique secret for every provisioned asset.